trade-finance-manager-ui/server/types/view-models/utilisation-report-reconciliation-for-report-view-model.ts

```diff
@@ -91,11 +91,11 @@ export type UtilisationReportReconciliationForReportViewModel = BaseViewModel &
   bank: SessionBank;
   formattedReportPeriod: string;
   reportId: string;
-  enablePaymentsReceivedSorting: boolean;
-  feeRecordPaymentGroups: FeeRecordPaymentGroupViewModelItem[];
+  facilityIdQuery?: string;
   tableDataError?: ErrorSummaryViewModel;
   filterError?: ErrorSummaryViewModel;
-  facilityIdQuery?: string;
+  enablePaymentsReceivedSorting: boolean;
+  premiumPayments: FeeRecordPaymentGroupViewModelItem[];
```

Comment:
Would it make sense to also update the name of this type? Perhaps, it should become`PremiumPaymentViewModel[]`?

Reply:
Updated